mod/wikirate/set/right/my_contributions.rb

```diff
@@ -47,15 +57,19 @@ def sections
   end
 
   view :contribution_counts do |args|
-    content_tag :div, :class=>'counts' do
-      card.sections.map do |count, name, contr_name|
-        content_tag :a, :class=>"item", :href=>"##{contr_name}" do
-          %{
-            <span class="#{name.downcase}">#{count}</span>
-            <p class="legend">#{name}</p>
-          }.html_safe
-        end
-      end.join("\n")
+    content_tag :div, class: 'counts' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.